src/ucp/core/ucp_context.c

```diff
@@ -314,6 +314,12 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "another thread, or incoming active messages, but consumes more resources.",
    ucs_offsetof(ucp_context_config_t, flush_worker_eps), UCS_CONFIG_TYPE_BOOL},
 
+  {"FENCE_WORKER_EPS", "auto",
+   "Enable flushing the worker by fencing its endpoints. Allows completing\n"
+   "the fence operation in a bounded time even if there are new requests on\n"
+   "another thread, or incoming active messages, but consumes more resources.",
```

Comment:
it does not affect flush, i'd rephrase it, like:
`Enable strong sync mode for ucp_worker_fence. By default it is used when multiple lanes are used for RMA operations.`